docs/ref/django-admin.txt

```diff
@@ -1467,10 +1467,12 @@ Enables :ref:`SQL logging <django-db-logger>` for failing tests. If
 Runs tests in separate parallel processes. Since modern processors have
 multiple cores, this allows running tests significantly faster.
 
-By default ``--parallel`` runs one process per core according to
-:func:`multiprocessing.cpu_count()`. You can adjust the number of processes
-either by providing it as the option's value, e.g. ``--parallel=4``, or by
-setting the :envvar:`DJANGO_TEST_PROCESSES` environment variable.
+Using ``--parallel`` without a value, or with the value ``auto``, runs one test
+process per core according to :func:`multiprocessing.cpu_count()`. You can
+override this by passing the desired number of processes, e.g.
+``--parallel 4``. You can also enable ``--parallel`` without passing the flag
+by setting the :envvar:`DJANGO_TEST_PROCESSES` environment variable to the
+desired number of processes.
```

Comment:
We can restore the previous version after fixing https://github.com/django/django/pull/14614#discussion_r666678802:
```suggestion
``--parallel 4``, or by setting the :envvar:`DJANGO_TEST_PROCESSES` environment
variable.
```

Reply:
I think the previous wording is unclear - it sounds like you still need to pass `--parallel` when setting the env var.